digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -2264,18 +2274,39 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
         mapLightLane(usedService.getByZoomLevel( Some(LinkGeomSource.NormalLinkInterface)))
       } else {
         validateBoundingBox(boundingRectangle)
-        val assets = usedService.getByBoundingBox(boundingRectangle)
+        val (assets, roadLinksWithoutLanes) = usedService.getByBoundingBox(boundingRectangle, withWalkingCycling = params("withWalkingCycling").toBoolean)
 
         val updatedInfo = roadAddressService.laneWithRoadAddress(assets)
         val frozenInfo = roadAddressService.experimentalLaneWithRoadAddress( updatedInfo.map(_.filterNot(_.attributes.contains("VIITE_ROAD_NUMBER"))))
-        mapLanes(updatedInfo ++ frozenInfo)
+        mapLanes(updatedInfo ++ frozenInfo) ++ roadLinksWithoutLanes.map(link => Seq(roadLinkToApi(link, false)))
       }
 
     } getOrElse {
       BadRequest("Missing mandatory 'bbox' parameter")
     }
   }
 
+  get("/lanes/viewOnlyLanes") {
+    params.get("bbox").map { bbox =>
+      val boundingRectangle = constructBoundingRectangle(bbox)
+      validateBoundingBox(boundingRectangle)
+      val viewOnlyLanes = laneService.getViewOnlyByBoundingBox(boundingRectangle, withWalkingCycling = params("withWalkingCycling").toBoolean)
```

Comment:
Shouldn't we do a getOrElse instead of a direct  params("withWalkingCycling").toBoolean ?

Reply:
Done like this params.getAsOrElse[Boolean]("withWalkingCycling", false)